pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go

```diff
@@ -167,8 +167,8 @@ func TestNodeResourcesLeastAllocated(t *testing.T) {
 		},
 		{
 			// Node1 scores on 0-10 scale
-			// CPU Score: ((10000 - 6000) *10) / 10000 = 40
-			// Memory Score: ((20000 - 5000) *10) / 20000 = 75
+			// CPU Score: ((10000 - 6000) *100) / 10000 = 40
```

Comment:
Can you say MaxNodeScore instead? like the other comment.

Reply:
sure, sure!